manager/logbroker/broker.go

```diff
@@ -260,13 +271,16 @@ func (lb *LogBroker) nodeConnected(nodeID string) {
 	lb.mu.Lock()
 	defer lb.mu.Unlock()
 
-	lb.connectedNodes[nodeID] = struct{}{}
+	lb.connectedNodes[nodeID] = make(map[*subscription]struct{})
```

Comment:
We shouldn't hit that condition because that would mean a node had called ListenSubscriptions twice and that shouldn't happen, but I understand that it is important to guard against the possibility of errors.